benchmarks/auto_benchmark.py

```diff
@@ -277,14 +278,23 @@ def main():
         action="store",
         help="true: skip torchserve installation. default: true",
     )
+    parser.add_argument(
```

Comment:
should this default to true? The official benchmark reports seems like teh exception

Reply:
I'm okay either way. The command doesn't error out if upload fails. So, false is fine.